### DISCUSSION TITLE
Config.c for BME280 Aprs

### DISCUSSION THREAD

**Discussion by linux150** [mikaelnousiainen/RS41ng#47] [Q&A]
Can anybody help a newbie configuring the config.c to put out the values from the BME280 vis aprs?
73 Martin

**Comment by linux150** [comment]
Ok, checked my wiring twice, sensor is correct wired. 3,3V are on the sensor but all i get is
`APZ41N de GW2151-13 via WIDE1-1 WIDE2-1 Ctl C UI Pid=F0 Len=88>
/000216z0000.00S/00000.00W_.../...g...t032h01b00000 0C 0% 0mb PC 0 RI 0 uR/h - WX GW2151`

Maybe any settings wrong in config.h and config.c ?!
Attached them, maybe somebody can take a look ?
73
[config.zip](https://github.com/mikaelnousiainen/RS41ng/files/12036846/config.zip)

**Reply by mikaelnousiainen** [reply]
Did you get this working? Still having the same issue?

**Comment by linux150** [comment]
HiNo gave up, learning now for my license and than after getting my ham call maybe Test again 😉73MartinAm 08.08.2023 um 17:38 schrieb Mikael Nousiainen ***@***.***>:﻿
Did you get this working? Still having the same issue?
